src/main/telemetry/smartport.c

```diff
@@ -85,15 +110,57 @@ enum
     FSSP_DATAID_HOME_DIST  = 0x0420 ,
     FSSP_DATAID_GPS_ALT    = 0x0820 ,
     FSSP_DATAID_ASPD       = 0x0A00 ,
+    FSSP_DATAID_TEMP       = 0x0B70 ,
+    FSSP_DATAID_TEMP1      = 0x0B71 ,
+    FSSP_DATAID_TEMP2      = 0x0B72 ,
+    FSSP_DATAID_TEMP3      = 0x0B73 ,
+    FSSP_DATAID_TEMP4      = 0x0B74 ,
+    FSSP_DATAID_TEMP5      = 0x0B75 ,
+    FSSP_DATAID_TEMP6      = 0x0B76 ,
+    FSSP_DATAID_TEMP7      = 0x0B77 ,
+    FSSP_DATAID_TEMP8      = 0x0B78 ,
     FSSP_DATAID_A3         = 0x0900 ,
     FSSP_DATAID_A4         = 0x0910
 };
 
 const uint16_t frSkyDataIdTable[] = {
     FSSP_DATAID_SPEED     ,
     FSSP_DATAID_VFAS      ,
+    FSSP_DATAID_VFAS1     ,
```

Comment:
These entries should be conditional to `USE_ESC_SENSOR`, to avoid flash space wastage for targets without ESC sensor support.